data/net/minecraft/world/item/DyeableHorseArmorItem.mapping

```diff
@@ -0,0 +1,6 @@
+CLASS net/minecraft/world/item/DyeableHorseArmorItem
+	COMMENT Constructor for the DyeableHorseArmor
+	METHOD <init> (ILjava/lang/String;Lnet/minecraft/world/item/Item$Properties;)V
+			COMMENT The given protection level of the {@code HorseArmorItem}
+			COMMENT the texture path identifier for the {@code DyeableHorseArmorItem}, @see {@code HorseArmorItem} for further information.
```

Comment:
I don't think this `@see` is used correctly? Either:
- no javadoc tags, or `@code` tags for the class name.
- a separate line with `@see` and the FQN
- an inline `{@link }` with the FQN